frontend/src/action-creator.async.js

```diff
@@ -850,17 +850,19 @@ export const getGuestUploadInfo = token => dispatch => {
   })
 }
 
-const eventTypesParam = '&exclude_event_types=' + global.GLOBAL_excludedNotifications
+const defaultExcludedEventTypesParam = '&exclude_event_types=' + global.GLOBAL_excludedNotifications
 
 export const getNotificationList = (
   userId,
   {
     excludeAuthorId = null,
     notificationsPerPage = NUMBER_RESULTS_BY_PAGE,
     nextPageToken = null,
-    workspaceId = null
+    workspaceId = null,
+    allEvents = false
   }) => async dispatch => {
-  const queryParameterList = [eventTypesParam]
+  const queryParameterList = []
+  if (!allEvents) queryParameterList.push(defaultExcludedEventTypesParam)
```

Comment:
err… I think that the need for the activities query is to have the same list of exclusions as the notification wall query apart from `content.modified`.

Reply:
fixed this